ftplugin/vader.vim

```diff
@@ -49,3 +56,9 @@ augroup vader_syntax
   " autocmd FileType <buffer> call vader#syntax#include(1, '$')
 augroup END
 
+if exists('b:undo_ftplugin')
+  let s:undo_ftplugin = b:undo_ftplugin.' | '.s:undo_ftplugin
+endif
+let b:undo_ftplugin = 'setl sw< ts< sts< et< cms< isk<'
+      \ . ' | au! vader_syntax * <buffer>'
+      \ . ' | unlet b:vader_label b:vader_eos | ' . s:undo_ftplugin
```

Comment:
The problem was that the `unlet` was added as an autocommand for `*`..
Wrapped it in `exe` now.